### ISSUE TITLE
Don’t Cling to Things

### ISSUE THREAD

**Issue by JattMannu** [JattMannu/react-blog-github#34]
## Endowment Effect

The BMW gleamed in the parking lot of the used-car dealership. Although it had a few miles on the odometer, it looked in perfect condition. I know a little about used cars, and to me, it was worth around $40,000. However, the salesman was pushing for $50,000 and wouldn’t budge a dime. When he called the next week to say he would accept $40,000 after all, I went for it. The next day, I took it out for a spin and stopped at a gas station. The owner came out to admire the car—and proceeded to offer me $53,000 in cash on the spot. I politely declined. Only on the way home did I realize how ridiculous I was to have said no. Something that I considered worth $40,000 had passed into my possession and suddenly taken on a value of more than $53,000. If I were thinking purely rationally, I would have sold the car immediately. But, alas, I’d fallen under the influence of the endowment effect. We consider things to be more valuable the moment we own them. In other words, if we are selling something, we charge more for it than what we ourselves would be willing to spend.

To probe this, psychologist Dan Ariely conducted the following experiment: In one of his classes, he raffled tickets to a major basketball game, then polled the students to see how much they thought the tickets were worth. The empty-handed students estimated around $170, whereas the winning students would not sell it below an average of $2,400. The simple fact of ownership makes us add zeros to the selling price.

In real estate, the endowment effect is palpable. Sellers become emotionally attached to their houses and thus systematically overestimate their value. They balk at the market price, expecting buyers to pay more—which is completely absurd since this excess is little more than sentimental value.

Richard Thaler performed an interesting classroom experiment at Cornell University to measure the endowment effect. He distributed coffee mugs to half of the students and told them they could either take the mug home or sell it at a price they could specify. The other half of the students who didn’t get a mug were asked how much they would be willing to pay for a mug. In other words, Thaler set up a market for coffee mugs. One would expect that roughly 50 percent of the students would be willing to trade—to either sell or buy a mug. But the result was much lower than that. Why? Because the average owner would not sell below $5.25, and the average buyer would not pay more than $2.25 for a mug.

We can safely say that we are better at collecting things than at casting them off. Not only does this explain why we fill our homes with junk, but also why lovers of stamps, watches, and pieces of art part with them so seldomly.

Amazingly, the endowment effect affects not only possession, but also near ownership. Auction houses like Christie’s and Sotheby’s thrive on this. A person who bids until the end of an auction gets the feeling that the object is practically theirs, thus increasing its value. The would-be owner is suddenly willing to pay much more than planned, and any withdrawal from the bidding is perceived as a loss—which defies all logic. In large auctions, such as those for mining rights or mobile radio frequencies, we often observe the winner’s curse: Here, the successful bidder turns out to be the economic loser when he gets caught up in the fervor and overbids. I’ll offer more insight on the winner’s curse in chapter 35.

There’s a similar effect in the job market. If you are applying for a job and don’t get a call back, you have every reason to be disappointed. However, if you make it to the final stages of the selection process and then receive the rejection, the disappointment can be much bigger—irrationally. Either you get the job or you don’t; nothing else should matter.

In conclusion: Don’t cling to things. Consider your property something that the “universe” (whatever you believe this to be) has bestowed to you temporarily. Keep in mind that it can recoup this (or more) in the blink of an eye.
